CHALLENGE 2.0/distance_calculator.py

```diff
@@ -0,0 +1,9 @@
+def calculate_distance():
+    x1=float(input("enter value here; "))
+    y1=float(input("enter value here; "))
+    x2 = float(input("enter value here; "))
+    y2=float(input("enter value here; "))
```

Comment:
Don't use input inside this function. Do something like 

```suggestion
def calculate_distance(x1, y1, x2, y2, d):
    ...
```